pkg/virt-controller/watch/vm.go

```diff
@@ -2401,3 +2403,19 @@ func (c *VMController) resolveControllerRef(namespace string, controllerRef *v1.
 	}
 	return vm.(*virtv1.VirtualMachine)
 }
+
+func autoAttachInputDevice(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) {
+	autoAttachInput := vm.Spec.Template.Spec.Domain.Devices.AutoattachInputDevice
+	// Default to False if nil and return
+	if autoAttachInput == nil || (autoAttachInput != nil && *autoAttachInput == false) {
```

Comment:
No need to check `autoAttachInput != nil`, because boolean operators use short-circuit. If the first operand of `||` is `true` then the second one is not evaluated, so it will not panic.

Reply:
Done.